apps/docs/content/guides/auth/auth-email-templates.mdx

```diff
@@ -27,7 +27,7 @@ The templating system provides the following variables for use:
 
 ## Editing email templates
 
-On self-hosted Supabase projects, edit your email templates on the [Email Templates](/dashboard/project/_/auth/templates) page. On self-hosted projects or in local development, edit your [configuration files](/docs/guides/cli/customizing-email-templates).
+On regular Supabase projects, edit your email templates on the [Email Templates](/dashboard/project/_/auth/templates) page. On self-hosted projects or in local development, edit your [configuration files](/docs/guides/cli/customizing-email-templates).
```

Comment:
maybe `hosted` rather than `regular`?

Reply:
@fsansalvadore I've changed it to regular.